### PR TITLE
Make sure you have resolution docs for your alerts.

### PR DESCRIPTION
**Pull request by unixorn**
Anything you care enough to page about should have a wiki describing how to
resolve it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ It's important to have the following:
 	* [Graphviz description language](http://www.graphviz.org/content/dot-language)
 * An overview of the company’s infrastructure.
 * A intranet/wiki or enterprise social network to Learn about different teams, key members and pictures. On day one, one can easily get overwhelmed with lots of new names and faces.
+* Have documentation for your alerts. If something is important enough to page the oncall person about, it's important enough to have a wiki entry about it. If you alert because _foo queue is too long_, there should be a wiki entry describing how to fix it.
```
Comment (theckman) [inline README.md]: I'm probably being a pedantic nerd, but can we call them `runbooks`?

>Have runbooks for your alerts.

Comment (unixorn) [inline README.md]: Sure. Give me 2 minutes.

**Comment by actionjack** [comment]
@unixorn I think I have this here: https://github.com/actionjack/so-you-want-to-onboard-a-devops-engineer/blame/master/README.md#L55 but feel free to replace that line with your text because it has more fidelity 👍

**Comment by actionjack** [comment]
Actually, I'll add it since it's subtly different. Thanks!

**Pull request merged**